src/lib.rs

```diff
@@ -868,13 +900,33 @@ pub async fn up(docker: &Docker, config: &Config) -> Result<(), Error> {
             docker.is_network_up(&network_name).await?,
             "The network should now be up"
         );
+    } else {
+        debug!("Network {} already exists", network_name);
     }
 
+    match config.up {
```

Comment:
What order has been changed here? Are we just making sure the docker network has been created before we run the up scripts?

Reply:
That's the idea, yes.